cmd/state-installer/cmd.go

```diff
@@ -355,7 +355,7 @@ func postInstallEvents(out output.Outputer, cfg *config.Instance, an analytics.D
 		cmd, args := exeutils.DecodeCmd(params.command)
 		if _, _, err := exeutils.ExecuteAndPipeStd(cmd, args, envSlice(binPath)); err != nil {
 			an.EventWithLabel(AnalyticsFunnelCat, "forward-command-err", err.Error())
-			return errs.Wrap(err, "Running provided command failed, error returned: %s", errs.JoinMessage(err))
+			return errs.Silence(err)
```

Comment:
This is dropping the error message it was adding, which is valuable as Go errors don't have stack traces.

I won't comment on the other instances, but it goes for all of them.